padle/padle.py

```diff
@@ -0,0 +1,801 @@
+            embed = discord.Embed(title="PADle #{} Stats".format(await self.config.num_days()), type="rich")
+            if completes == 0:
+                embed.description = (f"**Total Wins**: {completes}\n**Total Losses**: {giveups}\n**Win Rate**: 0%\n"
+                                     "**Average Guess Count**: 0")
+            else:
+                embed.description = ("**Total Wins**: {}\n**Total Losses**: {}\n**Win Rate**: {:.2%}\n"
+                                     "**Average Guess Count**: {:.2f}").format(completes, giveups,
+                                                                               completes / (completes + giveups),
+                                                                               (average / completes))
+            embed.set_footer(text=f"Try {ctx.prefix}padle globalstats [day] for other day's stats!",
+                             icon_url=TSUBAKI_FLOWER_ICON_URL)
+        else:
+            embed = await self.get_past_padle_embed(ctx, day)
+        await ctx.send(embed=embed)
+
+    @padle.command()
+    async def stats(self, ctx):
+        """Personal PADle stats"""
+        if await self.config.user(ctx.author).start() and not await self.config.user(ctx.author).done():
+            return await send_cancellation_message(ctx, "Please finish today's PADle before checking your stats!")
+        all_guesses = await self.config.user(ctx.author).all_guesses()
+        played = len(all_guesses)
+        save_daily = await self.config.save_daily_scores()
+        if played == 0:
+            return await send_cancellation_message(ctx, "You haven't played PADle yet!")
+        cur_streak = 0
+        max_streak = 0
+        wins = 0
+        for day in range(1, await self.config.num_days() + 1):
+            if day == await self.config.num_days():
+                correct_id = await self.config.padle_today()
+            else:
+                correct_id = save_daily[day - 1][0]
+            if str(day) in all_guesses and len(all_guesses[str(day)]) != 0 and all_guesses[str(day)][-1] == correct_id:
+                cur_streak += 1
+                wins += 1
+            else:
+                if cur_streak > max_streak:
+                    max_streak = cur_streak
+                if day != await self.config.num_days():
+                    cur_streak = 0
+
+        if cur_streak > max_streak:
+            max_streak = cur_streak
+        all_monsters_guessed = []
+        for key, value in all_guesses.items():
+            all_monsters_guessed.extend(value)
+        mode = max(set(all_monsters_guessed), key=all_monsters_guessed.count)
+        dbcog = await self.get_dbcog()
+        m = dbcog.get_monster(mode)
+        m_embed = EmbedView(
+            EmbedMain(
+                title=f"{ctx.author.name}'s PADle Stats",
+                description=(f"**Games Played**: {played}\n"
+                             "**Win Rate**: {:.2%}\n".format(wins / played) +
```

Comment:
You should use f strings for the whole block if you use it for one part. You can do `{wins / played:.2%}`